index.html

```diff
@@ -81,6 +81,51 @@ <h1>
 		</section>
 
 		<section class="works" id="portfolio">
+			<div class="popup-container">
+				<!-- TEMPLATE TO USE WITH JS -->
+				<!-- 	<div class="main-container">
+					<div class="content-container">
+						<div class="header">
+							<h1>title</h1>
+							<div id="btn-close" class="fas fa-times"></div>
+						</div>
+						<ul class="info">
+							<li>info</li>
+							<li>info</li>
+							<li>info</li>
+							<li>info</li>
+						</ul>
+						<div class="image-container">
+							<img
+								src="img/desktop/desktop-img1.jpg"
+								class="card-image"
+								alt=""
+							/>
+						</div>
+						<div class="container-secondary">
+							<p>
+								Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+								Reprehenderit, fuga?
+							</p>
+							<div class="skills-info">
+								<ul>
+									<li>html</li>
+									<li>css</li>
+									<li>javascript</li>
+								</ul>
+								<div class="buttons-container">
+									<button type="button">
+										see live <i class="fas fa-check"></i>
+									</button>
+									<button type="button">
+										see live <i class="fab fa-github"></i>
+									</button>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> -->
+			</div>
 			<!-- card1 -->
 			<div class="card">
 				<img src="img/desktop/desktop-img1.jpg" alt="desktop image" />
```

Comment:
https://github.com/LuisSalas94/My-Portfolio/blob/10cae6c3d5990eb3cdc699ca21445e1005420880/index.html#L129-L171

- [ ] As required :point_down: 

![image](https://user-images.githubusercontent.com/57421823/151015393-2368bdbc-0f28-46f4-9357-b17c3811e52d.png)

As seen above, all the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

This means you can use the `projects` array and logic that you already have, to populate the projects section of the main page entirely with javascript!

If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty